packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -738,6 +740,8 @@ const TagBox = (SelectBox as any).inherit({
       selectedItems: this.option('selectedItems'),
     };
 
+    const tagData = $tag.data(TAGBOX_TAG_DATA_KEY);
```

Comment:
What value of tagData here? What value will be set in aria-label if you do this?

Reply:
the content of tag data is also the innerHTML of the current multitag like for example
there are 4 items so a multitag will be created with a inner html called "**4 more**"
tag data = "**4 more**"

i used it so that nvda will utter / say "4 more tag "

Reply:
I see that in this case your var has underfunded value.

Reply:
actually nevermind my comment i also encountered the undefined variable its very weird one day ago it works as intended. anyway i have a fix for this one i just use args.text....

Reply:
[f086478](https://github.com/DevExpress/DevExtreme/pull/27063/commits/f086478a89c25c1071298b9fdace06ddbf2db9aa)